.gitignore

```diff
@@ -4,6 +4,7 @@ node_modules
 src/handbook/media
 src/docs/*
 !src/docs/docs.json
+src/blueprints/*
```

Comment:
This only works now because you've already put blueprints.njk and templates.njk are in Git already. Can we add:

```
!src/blueprints/*.njk
```